pipelines/tekton/azureml-container-pipeline/model-upload/local-model-pvc.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: local-model-pvc-template
+objects:
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    name: ${PVC}
+  spec:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: ${SIZE} #1Gi
+parameters:
+- name: PVC
+  description: Name of PVC to be mounted to pod
+  from: '[A-Z0-9]{8}'
+  generate: expression
+- name: SIZE
+  description: Size of the PVC
+  from: '[A-Z0-9]{8}'
+  generate: expression
```

The combination of `generate: expression` and `from` fields in each parameter definition here will cause a random value to be generated matching the `from` value, if no parameter is passed explicitly. I don't think this is appropriate for the `SIZE` parameter, and may not be for the `PVC` one either, but I'm less sure about that.

The following might be a better way to have more sane default/fallback values (which double as hints for the format of expected values, which I guess is the intention of the comment on L15):

```suggestion
- name: PVC
  description: Name of PVC to be mounted to pod
  value: local-model
- name: SIZE
  description: Size of the PVC
  value: 1Gi
```

---

Thanks I made that change 👍